### DISCUSSION TITLE
Error updating my CouchDB on Raspberry PI4

### DISCUSSION THREAD

**Discussion by steveross1975** [apache/couchdb#2956] [Q&A] [answered]
Hi guys @CouchDB, I'm getting a weird error updating my CouchDB on Raspberry PI4 

`Skipping acquire of configured file 'main/binary-armhf/Packages' as repository 'https://apache.bintray.com/couchdb-deb buster InRelease' doesn't support architecture 'armhf'`

Any guess? Thnx!

**Comment by wohali** [comment, marked as answer]
Hi @steveross1975 ,

CouchDB doesn't supply packages for 32-bit ARM at this time. That means that the default Raspbian OS install won't run our CouchDB packages.

For the Raspberry Pi 3 and 4, you can install [Ubuntu 20.04 (64-bit)](https://ubuntu.com/download/raspberry-pi) and then install our CouchDB packages.

**Reply by steveross1975** [reply]
Ok, I see...
Actually I don't remember how I've managed to install it in the first place (maybe a fork of the main project...), but all I know is that you definitely need to support arm architecture... CouchDB and RPI are the perfect match! 
Thanks a lot for your answer.
